tests/functional/adapter/test_incremental_predicates.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from dbt.tests.adapter.incremental.test_incremental_predicates import BaseIncrementalPredicates
+
+
+models__spark_incremental_predicates_sql = """
```

Comment:
Do we need to re-define this?

Reply:
i believe I needed to redefine this in my [dbt-spark PR](https://github.com/dbt-labs/dbt-spark/pull/436/files#diff-476253c5fe28ecc18fa10e0819037b5ffe7e877d781aa83199c3d42961a3615e), and I assumed I needed to do the same here. 

i did just run the SQL in databricks without the cast function and it seemed to execute. I can remove this if we don't think this is necessary!

Reply:
Shall we just rename it to `models__databricks_incremental_predicates_sql`, then?